styleguide/source/_patterns/05-pages/news-as-gated.twig

```diff
@@ -17,8 +17,10 @@
   {% set promo = promoAsInline %}
   {% include '@molecules/promo.twig' %}
   {% include '@atoms/paragraph.twig' %}
-  {% include '@molecules/tags.twig' %}
-  {% include '@organisms/four-up-teaser.twig' %}
+  <section class="ama__page--news__teasers">
```

Comment:
Can we label this something more generic and less tied to what is meant to be in this section. I suspect what is meant to be in this section may change over time. I suggest referring to it as "content bottom" or something else that refers more to it's location on the page than it's content.

Reply:
@labbydev I agree, but we're already calling the variant that appears at the bottom of the body content "Content Bottom" - do you have any other ideas?

Reply:
👍